aim/cli/convert/processors/wandb.py

```diff
@@ -0,0 +1,67 @@
+import click
+from aim import Run
+
+
+def parse_wandb_logs(repo_inst, entity, project, run_id):
+    try:
+        import wandb
+    except ImportError:
+        click.echo(
+            'Could not process wandb logs - failed to import "wandb" module.', err=True
+        )
+        return
+
+    client = wandb.Api()
+
+    if run_id is None:
+        # process all runs
+        runs = client.runs(entity + "/" + project)
+    else:
+        try:
+            # get the run by run_id
+            run = client.run(f"{entity}/{project}/{run_id}")
+        except Exception:
+            click.echo(f'Could not find run "{entity}/{project}/{run_id}"', err=True)
+            return
+        runs = (run,)
+
+    for run in runs:
+        if run.config.items():
+            aim_run = Run(
+                repo=repo_inst,
+                system_tracking_interval=None,
+                experiment=project
+            )
+            aim_run['wandb_run_id'] = run.id
+            aim_run['wandb_run_name'] = run.name
+            aim_run.description = run.notes
+
+            # Collect params & tags
+            aim_run['params'] = run.config
+            for tag in run.tags:
+                aim_run.add_tag(tag)
+
+            records = run.history()
+            keys = [key for key in run.history().keys()
+                    if not key.startswith('_')]
+
+            # Collect metrics
+            for step in list(records["_step"]):
+                record = records.loc[records["_step"] == step]
+                for key in keys:
```

Comment:
if I understand the API correctly; records are grouped by step and each key might correspond to different types (metric, artifact, etc.)? is there any guarantee of them being uniform? i.e., should same key for different steps have the same type?

Reply:
Exactly @alberttorosyan. I have just tried to insert different type of values under the same key and wandb doesn't throw an error. So the same key for different steps can have different types.